src/webgpu/api/operation/render_pipeline/culling_tests.spec.ts

```diff
@@ -0,0 +1,134 @@
+export const description = 'Test culling and rasterizaion state.';
```

Comment:
Can you add the test plan in the description similarly to https://github.com/gpuweb/cts/pull/223/files ?